cluster/addons/registry/registry.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta2
```

Comment:
do you intend to backport this config? For both the master branch and the 1.9 branch you should be able to use the v1 apps apis instead of the v1beta2 ones.

Reply:
@roberthbailey  I noticed this, I will modify it.